docs/lib/datastore/data-access.md

```diff
@@ -4,25 +4,41 @@ description: Learn how to save, query, paginate, update, delete and observe data
 ---
 <inline-fragment platform="js" src="~/lib/datastore/fragments/js/data-access/importing-datastore-snippet.md"></inline-fragment>
 
-## Save Data
+## Create and update
 
-To write any data to the DataStore you can pass an instance of a Model to `DataStore.save()` and it will be persisted in offline storage. At this point you can use it as an item in a normal data store such as querying, updating or deleting. If you choose to later connect to the cloud the item will be synchronized using GraphQL mutations and any other systems connected to the same backend can run queries or mutations on these items as well as observe them with GraphQL subscriptions.
+To write data to the DataStore you can pass an instance of a model to `Amplify.DataStore.save()`.
 
 <inline-fragment platform="js" src="~/lib/datastore/fragments/js/data-access/save-snippet.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/data-access/save-snippet.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/datastore/fragments/android/data-access/save-snippet.md"></inline-fragment>
 
+The `save` method takes care of creating a new record or updating an existing one in case the model `id` already exists in the database.
```

Comment:
```suggestion
`save()` will create a new object, if the model's ID is not already known to the DataStore. Otherwise, if there is an existing record with the same ID, `save()` will update it.
```